### PR TITLE
Implement admin-statistics mail & session API

### LINKED ISSUE #667: Implement admin API for mails & sessions API to admin route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Parent Issue #642
Work to be done
- Add models for mails & users
- Implement admin API for mails & users API to admin route

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Add admin-statistics-mail model
- Add admin-statistics-session model
- Implement admin-statistics mail & session API

Fixes #667

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-673-open-event-frontend-fossasia.surge.sh

File: app/routes/admin/index.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ export default Route.extend({
           op   : 'eq',
           val  : 1
         }
+      }),
+      mails: this.get('store').findRecord('admin-statistics-mail', {
```
Comment (geekyd) [inline app/routes/admin/index.js]: Please do not use findRecord leads Error 500, it requires an id.

Comment (CosmicCoder96) [inline app/routes/admin/index.js]: @geekyd Sorry left it by mistake while testing something 😅

**Comment by CosmicCoder96** [comment]
@geekyd Done!

**Comment by niranjan94** [comment]
Deployment link: https://pr-673-open-event-frontend-fossasia.surge.sh

**Pull request merged**